tests/utils.cpp

```diff
@@ -38,6 +39,9 @@ Tests::run(int argc, char** argv)
                 std::find(filter.begin(), filter.end(), test.first);
             run_test = filter_it != filter.end();
         }
+
+        std::cout << (run_test ? "Running" : "Skipping") << " test "
```

Comment:
The stdout/stderr output is only shown when running with `ctest -VV` or invoking the resulting test executable directly otherwise its suppressed. This will show the subtest running like the below:
```
Running test test_cons
Running test test_ranges
Running test test_clip_v1_to_v2_null
Running test test_clip_v1_to_v2
Running test test_clip_media_representation
```

I thought it was useful to add since when you get a failure you can run with the verbose option to know which sub-test failed.

Reply:
That definitely helps, I wonder if we should enable the verbose mode in the GitHub Actions.

Reply:
Best to open an issue, so someone with the requisite knowledge will see it.